system-variables.md

```diff
@@ -232,40 +245,43 @@ mysql> SELECT * FROM t1;
 ### tidb_broadcast_join_threshold_count <span class="version-mark">New in v5.0</span>
 
 - Scope: SESSION | GLOBAL
-- Default value: 10240
+- Default value: `10240`
+- Range: `[0, 9223372036854775807]`
 - The unit of the variable is rows. If the objects of the join operation belong to a subquery, the optimizer cannot estimate the size of the subquery result set. In this situation, the size is determined by the number of rows in the result set. If the estimated number of rows in the subquery is less than the value of this variable, the Broadcast Hash Join algorithm is used. Otherwise, the Shuffled Hash Join algorithm is used.
 
 ### tidb_broadcast_join_threshold_size <span class="version-mark">New in v5.0</span>
 
 - Scope: SESSION | GLOBAL
-- Default value: 104857600 (which equals 100 megabytes)
-- If the table size (in the unit of bytes) is less than the value of the variable, the Broadcast Hash Join algorithm is used. Otherwise, the Shuffled Hash Join algorithm is used.
+- Default value: `104857600`
+- Range: `[0, 9223372036854775807]`
+- Unit: Bytes
+- If the table size is less than the value of the variable, the Broadcast Hash Join algorithm is used. Otherwise, the Shuffled Hash Join algorithm is used.
 
 ### tidb_build_stats_concurrency
 
-- Scope: SESSION
-- Default value: 4
+- Scope: SESSION | GLOBAL
+- Default value: `4`
 - This variable is used to set the concurrency of executing the `ANALYZE` statement.
 - When the variable is set to a larger value, the execution performance of other queries is affected.
 
 ### tidb_capture_plan_baselines <span class="version-mark">New in v4.0</span>
 
 - Scope: SESSION | GLOBAL
-- Default value: OFF
+- Default value: `OFF`
 - This variable is used to control whether to enable the [baseline capturing](/sql-plan-management.md#baseline-capturing) feature. This feature depends on the statement summary, so you need to enable the statement summary before you use baseline capturing.
 - After this feature is enabled, the historical SQL statements in the statement summary are traversed periodically, and bindings are automatically created for SQL statements that appear at least twice.
 
 ### tidb_check_mb4_value_in_utf8
```

Comment:
Maybe this should link to `tidb_skip_utf8_check` and/or explain how these are related?

Reply:
Thanks! I will do it in a separate PR. For this I'm aiming for parity through an auto-generator.